mubench.reviewsite/src/Controller/TagController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace MuBench\ReviewSite\Controller;
+
+use Monolog\Logger;
+use MuBench\ReviewSite\DBConnection;
+
+class TagController
+{
+
+    private $db;
+    private $logger;
+
+    function __construct(DBConnection $db, Logger $logger)
+    {
+        $this->db = $db;
+        $this->logger = $logger;
+    }
+
+    public function saveTagForMisuse($misuse)
+    {
+        $tag = $this->getOrCreateTag($misuse['tag']);
+        $tag_id = $tag['id'];
+        $experiment = $misuse['exp'];
+        $detector = $misuse['detector'];
+        $project = $misuse['project'];
+        $version = $misuse['version'];
+        $misuse_id = $misuse['misuse'];
+        $this->logger->info("saving tag $tag_id for $experiment, $detector, " .
+            $project .
+            ", " .
+            $version .
+            ", " .
+            $misuse_id);
```

Comment:
Why do some inline variables and some explicit concatenations? Inline all. Make it a one-liner.